tests/core/test_test.py

```diff
@@ -1034,7 +1034,7 @@ def test_create_external_model_fixture(sushi_context: Context, mocker: MockerFix
     )
     _check_successful_or_raise(test.run())
 
-    assert len(test._fixture_table_cache) == len(sushi_context.models) + 1
+    assert len(test._fixture_table_cache) == 1
```

Comment:
This changed because we used to have `self.models.keys() | self.body.get("inputs", {})`, but I believe `self.models` is redundant here so I went ahead and removed it.